.changeset/pink-rocks-bake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@razorpay/blade": patch
+---
+
+feat(TextArea): Tagged Inputs
+feat(Tag): max-width is removed from Tag component
```

Comment:
Should we make this a minor bump since there are changes to tag as well as new prop additions?